tests/cases/standalone/order/order_variable_size_payload.result

```diff
@@ -137,7 +137,7 @@ SELECT * FROM tpch_q1_agg ORDER BY l_returnflag, l_linestatus;
 | R            | F            | 3785523 | 5337950526.47  | 5071818532.942  | 5274405503.049367  | 25.5259438574251   | 35994.029214030925 | 0.04998927856184382 | 148301      | 2 |
 +--------------+--------------+---------+----------------+-----------------+--------------------+--------------------+--------------------+---------------------+-------------+---+
 
-create table test5 (i int, s varchar, t BIGINT TIME INDEX);
+create table order_variable_size_payload_test5 (i int, s varchar, t BIGINT TIME INDEX);
```

Comment:
I remember that some test cases are ported from other databases. Do we need to keep the table name unchanged? @killme2008

Reply:
The change doesn't matter, but some of them are unnecessary I suppose.